package.json

```diff
@@ -63,28 +63,29 @@
         "identity-obj-proxy": "^3.0.0",
         "jest": "^26.4.2",
         "jest-canvas-mock": "^2.1.1",
-        "mini-css-extract-plugin": "^0.9.0",
+        "mini-css-extract-plugin": "^1.5.0",
         "mobx": "^4.0.0",
         "mobx-react": "^5.0.0",
         "moment-timezone": "^0.5.14",
         "optimize-css-assets-webpack-plugin": "^5.0.3",
-        "postcss-calc": "^7.0.1",
+        "postcss": "^7.0.35",
```

Comment:
postcss 8 requires postcss-loader 4 but first ckeditor would here need to update to the newest version: https://github.com/ckeditor/ckeditor5-dev/pull/695

Reply:
😢